src/addons/sourcemod/scripting/zr/infect.inc

```diff
@@ -201,6 +201,31 @@ void InfectOnCommandsCreate()
     RegConsoleCmd("zr_human", InfectHumanCommand, "Turn a client into a human. Usage: zr_human <filter> [respawn - 1/0]");
 }
 
+/**
+ * Calculates the weight of a player for mother zombie selection.
+ * Higher weight means lower chance of being selected.
```

Comment:
At first I thought more weight means more chances of being mz, maybe make it this way I find it a bit confusing

Reply:
Like this ?

Reply:
perfect!